### ISSUE TITLE
roberta权重参数文件生成不了

### ISSUE THREAD

**Issue by ODDHOOD** [wjx-git/IllegalTextDetection#5]
你好 好像这个训练数据集太小 还是什么原因 
                if global_step % args.eval_steps == 0:
                    logger.info('xx')
                    labels, preds, eval_loss = evaluate(args, dev_dataset, model, mode='dev')
                    writer.add_scalar('eval_loss', eval_loss, global_step=global_step)
                    matrix = classification_report(labels, preds, digits=4)
                    logger.info('*********** Eval Results ***********')
                    logger.info(matrix)
                if global_step % args.save_steps == 0:
                    save_model(args, model, tokenizer, global_step)
一直进不到这个代码块里 用的都是你的参数 所以生成不了权重文件 希望博主能给出解决方案 谢谢你
08/04/2023 21:14:43 - INFO - __main__ -   ***** Running training *****
08/04/2023 21:14:43 - INFO - __main__ -     Num examples = 100
08/04/2023 21:14:43 - INFO - __main__ -     Num Epochs = 4
08/04/2023 21:14:43 - INFO - __main__ -     Total train batch size = 16
08/04/2023 21:14:43 - INFO - __main__ -     Gradient Accumulation steps = 1
08/04/2023 21:14:43 - INFO - __main__ -     Total optimization steps = 28
08/04/2023 21:14:43 - INFO - __main__ -     Logging steps = 500
08/04/2023 21:14:43 - INFO - __main__ -     Save steps = 1500
只会训练 不会进到验证那部分

**Comment by wjx-git** [comment]
增加训练数据量，或者save_steps设为10， Logging steps = 10

类似的问题：
https://github.com/wjx-git/IllegalTextDetection/issues/1

**Comment by weeeeeesterly** [comment]
我还在为Num examples = 100....的事发愁，每一条训练打印只有2到7个内容，
然后把dataset打印出来了，长这样：400条tensor ,每一条有150个数字
(tensor([ 101, 2802,  872, 7716, 1239, 3419, 3683,  801, 3647, 3124, 2424, 4638,
        3952, 2767, 8013,  102,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,
           0,    0,    0,    0,    0,    0]),...
之后我调整了max_seq_len默认值为100，随即打印出的Num examples = 24978。
所以这个最大单条文本输入max_seq_len怎么设是合理的？

**Comment by wjx-git** [comment]
> 我还在为Num examples = 100....的事发愁，每一条训练打印只有2到7个内容， 然后把dataset打印出来了，长这样：400条tensor ,每一条有150个数字 (tensor([ 101, 2802, 872, 7716, 1239, 3419, 3683, 801, 3647, 3124, 2424, 4638, 3952, 2767, 8013, 102, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]),... 之后我调整了max_seq_len默认值为100，随即打印出的Num examples = 24978。 所以这个最大单条文本输入max_seq_len怎么设是合理的？

max_seq_len设置为你训练集里最长样本的长度即可。max_seq_len 和 num examples 没有关系。
